src/user/controllers/user.controller.ts

```diff
@@ -30,6 +33,7 @@ export class UserController {
     return this.users.getUsersWithParams(params);
   }
 
+  @UseGuards(CheckTokenGuard)
```

Comment:
recommended: as it was discussed on the task issuing meeting the route for creating new user better to set public (without guard)